tests/features/search/querying.feature

```diff
@@ -31,43 +31,51 @@ Feature: Search querying
     When I visit a `search page`
     And I enter "paris" in the `search box`
     And I click the `search button`
-    And I wait 2 seconds
     And I click a `search result`
     Then I see a `record page`
     And I don't see "paris" in the `search box`
 
-  Scenario: Using auto suggestion with keyboard populates search field
-    When I visit a `search page`
+  Scenario: No auto suggestion
+    Given I am on the `search page`
     And I enter "World" in the `search box`
     And I wait 1 second
```

Comment:
let's up this to 2 seconds. It's passing even though autosuggest is switched on, just because it's taking longer than a second to come in.